.github/workflows/core-ci.yml

```diff
@@ -73,7 +73,7 @@ jobs:
             pip install statsmodels tsfresh
           fi
           virtualenv testenv && source testenv/bin/activate
-          pip install -r requirements.txt && pip install pytest pytest-timeout
+          pip install -r requirements.txt && pip install -r requirements-extra.txt && pip install pytest pytest-timeout
```

Comment:
This group of tests if used to test if Mars can work under minimal requirements. Thus no extra requirements should be installed.

Reply:
`bokeh` is needed when import `service` api. Or can we add `bokeh` to `requirement.txt`

Reply:
Then try to make it optional.